src/test/java/org/jabref/gui/exporter/SaveDatabaseActionTest.java

```diff
@@ -27,8 +27,8 @@
 
 class SaveDatabaseActionTest {
 
-    private static final String TEST_FILE_PATH = "C:\\Users\\John_Doe\\Jabref";
-    private final File file = new File(TEST_FILE_PATH);
+    private static final String TEST_BIBTEX_LIBRARY_LOCATION = "C:\\Users\\John_Doe\\Jabref\\literature.bib";
+    private final File file = new File(TEST_BIBTEX_LIBRARY_LOCATION);
```

Comment:
Please change this to Path.get() and use nio.Path in the other instances as well

Reply:
Then the path variable can be removed as well.